OurScript.R

```diff
@@ -0,0 +1,196 @@
+## Loading Necessary Libraries ----
+library(tidyverse)
+library(skimr)
+library(visdat)
+library(ggplot2)
+
```

Comment:
General comment: We are unsure what the overall question is that motivated the EDA. The questions being asked here are implied but not stated.